12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,9 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+"""
+excersize #1
+Print list of all English lowercase letters
+"""
+
+#x = range(ord('a'),ord('z')+1)
+lowercase_letters = [x for x in range(256) if x >= ord('a') and x <= ord('z')]

```

Comment:
Prefer to use `ord('a')` and `ord('z')` as your limits in range, so you don't need the if